### PR TITLE
Integration name added to the integration-related activities

### PR DESCRIPTION
**Pull request by Yumfriez**

### CODE REVIEW

File: src/main/java/com/epam/ta/reportportal/core/events/activity/IntegrationCreatedEvent.java [hunk 1]
```diff
@@ -56,6 +65,7 @@ public Activity toActivity() {
 				.addObjectId(integrationActivityResource.getId())
 				.addObjectName(integrationActivityResource.getTypeName())
 				.addProjectId(integrationActivityResource.getProjectId())
+				.addHistoryField(Optional.of(integrationNameField))
```
Comment (pbortnik) [inline src/main/java/com/epam/ta/reportportal/core/events/activity/IntegrationCreatedEvent.java]: can be used 'addHistoryField(String field, String before, String after)' here, so we can remove upper above

File: src/main/java/com/epam/ta/reportportal/core/events/activity/IntegrationDeletedEvent.java [hunk 2]
```diff
@@ -56,6 +65,7 @@ public Activity toActivity() {
 				.addObjectId(integrationActivityResource.getId())
 				.addObjectName(integrationActivityResource.getTypeName())
 				.addProjectId(integrationActivityResource.getProjectId())
+				.addHistoryField(Optional.of(integrationNameField))
```
Comment (pbortnik) [inline src/main/java/com/epam/ta/reportportal/core/events/activity/IntegrationDeletedEvent.java]: can be used 'addHistoryField(String field, String before, String after)' here, so we can remove upper above

File: src/main/java/com/epam/ta/reportportal/core/events/activity/IntegrationCreatedEvent.java (hunk 1 above)
Comment (Yumfriez) [reply src/main/java/com/epam/ta/reportportal/core/events/activity/IntegrationCreatedEvent.java]: fixed

File: src/main/java/com/epam/ta/reportportal/core/events/activity/IntegrationDeletedEvent.java (hunk 2 above)
Comment (Yumfriez) [reply src/main/java/com/epam/ta/reportportal/core/events/activity/IntegrationDeletedEvent.java]: fixed

**Pull request merged**